docs/schema-conversion-logic.md

```diff
@@ -0,0 +1,98 @@
+# Schema conversion logic
+
+Metrictank comes with a schema conversion logic (`mdata/importer/converter.go`) which can be used to convert data from one storage schema into another one. This is currently only used by the whisper importer, but may in the future also be used to import from other types of data sources.
```

Comment:
Minor, but "Metrictank comes with a schema conversion logic" -> "Metrictank comes with schema conversion logic"

Pretty sure that usage is a [non-countable noun](https://grammar.yourdictionary.com/grammar/nouns/countable-and-non-countable-nouns.html)